.github/workflows/codeql-analysis.yml

```
@@ -6,56 +6,80 @@ on:
   pull_request:
     branches: [master, develop]
   schedule:
-    #        ┌───────────── minute (0 - 59)
-    #        │  ┌───────────── hour (0 - 23)
-    #        │  │ ┌───────────── day of the month (1 - 31)
-    #        │  │ │ ┌───────────── month (1 - 12 or JAN-DEC)
-    #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        │  │ │ │ │
-    #        *  * * * *
-    - cron: '30 1 * * 0'
+    - cron: 30 1 * * 0 # Runs every Sunday 1:30 am UTC
 
 jobs:
-  CodeQL-Build:
-    # CodeQL runs on ubuntu-latest, windows-latest, and macos-latest
+  CodeQL-Javascript:
     runs-on: ubuntu-latest
-
     permissions:
-      # required for all workflows
       security-events: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

We probably should use `v3` here.
```suggestion
        uses: actions/checkout@v3
```

---

Changed it! Thanks!